### PR TITLE
fix qunit tests - dateBox list strategy should create new dates correctly

### PR DESCRIPTION
**Pull request by churkin**

### CODE REVIEW

**Review by jsdmitry — APPROVED** [review]

File: js/ui/date_box/ui.date_box.strategy.list.js [hunk 1]
```diff
@@ -238,22 +237,28 @@ var ListStrategy = DateBoxStrategy.inherit({
...
+            date.setHours(hours);
+            date.setMinutes(minutes);
+            date.setSeconds(seconds);
+            date.setFullYear(year);
+            date.setMonth(month);
+            date.setDate(day);
         } else {
-            date = new Date();
-            uiDateUtils.normalizeTime(date);
+            date = new Date(year, month, day, hours, minutes, 0, 0);
```
Comment (San4es) [inline js/ui/date_box/ui.date_box.strategy.list.js:259]: Is this change necessary to fix the test?

**Review by San4es — APPROVED** [review]
LGTM
